### ISSUE TITLE
Unranked entries

### ISSUE THREAD

**Issue by HuvaaKoodia** [ludumdare/ludumdare#1555]
Currently you can disable some of the rating categories for your entry. How about all of them?

Alternatively there could be an unranked category alongside Compo, Jam and Unfinished. Alakajam has this system for one.

**Comment by mikekasprzak** [comment]
The categories you can't disable are `Overall`, `Fun`, `Innovation`, and `Theme`, all things that any valid entry should be capable of being rated in. If not, the game should be marked as Unfinished.

**Comment by HuvaaKoodia** [comment]
Ok. Feels a bit silly submitting a finished entry as unfinished, but I guess it will do for the time being. Thanks.

**Comment by mikekasprzak** [comment]
I'll just can't think of an example where you would want a legitimately
submitted game to not be judged as a game.

**Comment by HuvaaKoodia** [comment]
For me this is the case: I care about feedback and comments, not scores. I give out very honest and blunt criticism with public ratings (which reflect the criticism) and at times there is a nagging feeling in the back of my head, which I don’t exactly like: how are people rating my entry due to the criticism?

The hope is that after receiving such criticism those people who do check out my entry will see that there are no ratings and then either leave, or give it ago and write a proper comment (positive, negative, it is all good as long as constructive) in response.

While **I** don't care about the ratings, **others** do. If the grades do get bombed unfairly then people looking at my entry in the future are going to see the lower scores and potentially think: "Not worth my time!". 

I have no proof that rate bombing (or revenge voting) does happen, but the uncertainty annoys me to no end. The average grades are even less valuable information due to the possibility. Only honest constructive comments matter in the end. That's how I learn anything from the event.

In conclusion, probably not a widely used feature, but I would use it at least.
